variables.tf

```diff
@@ -1,12 +1,51 @@
-# variable "variable_name" {
-#	  description = "variable description"
-#   type        = number
-#   default     = 1
-# }
-
-# variable "variable_password" {
-#	  description = "variable description"
-#   type        = string
-#   sensitive   = true
-#   default     = "abc"
-# }
+variable "name" {
+  description = "The Load Balancer name"
+  type        = string
+}
+
+variable "region" {
+  description = "The region to start in"
+  type        = string
+}
+
+variable "size" {
+  description = "The size of the Load Balancer. It must be either lb-small, lb-medium, or lb-large. Defaults to lb-small. Only one of size or size_unit may be provided."
+  type        = string
+  default     = "lb-small"
+}
+
+variable "size_unit" {
+  description = "The size of the Load Balancer. It must be in the range (1, 100). Defaults to 1. Only one of size or size_unit may be provided."
+  type        = number
+  default     = 1
+}
+
+variable "vpc_uuid" {
+  description = "The ID of the VPC where the load balancer will be located."
+  type        = string
+  default     = ""
```

Comment:
Either null, as "real" empty value, or no default at all. Lets set it to null, so it's the same as in droplets and k8s.

Reply:
ok